indico/modules/events/registration/controllers/management/invitations.py

```diff
@@ -62,10 +88,79 @@ def _process(self):
             flash(ngettext('The invitation has been sent.',
                            '{n} invitations have been sent.',
                            num).format(n=num), 'success')
-            return jsonify_data(invitation_list=_render_invitation_list(self.regform))
+            return jsonify_data(**_get_invitation_data(self.regform))
         return jsonify_template('events/registration/management/regform_invite.html', regform=self.regform, form=form)
 
 
+class RHRegistrationFormRemindersSend(RHManageRegFormBase):
+    """Send a reminder to pending invitees."""
+
+    @use_kwargs({
+        'from_address': fields.String(required=True, validate=not_empty),
+        'body': fields.String(required=True, validate=not_empty),
+        'subject': fields.String(required=True, validate=not_empty),
+        'bcc_addresses': fields.List(LowercaseString(validate=validate.Email())),
+        'copy_for_sender': fields.Bool(load_default=False),
+    })
+    def _process(self, from_address, body, subject, bcc_addresses, copy_for_sender):
+        if from_address not in self.event.get_allowed_sender_emails():
+            abort(422, messages={'from_address': ['Invalid sender address']})
+        invitations = _query_pending_invitations(self.regform)
+        for invitation in invitations:
+            email_body = replace_placeholders('registration-reminder-email', body, event=self.event,
+                                              invitation=invitation)
+            email_subject = replace_placeholders('registration-reminder-email', subject,
+                                                 event=self.event, invitation=invitation)
+            tpl = get_template_module('emails/custom.html', subject=email_subject, body=email_body)
```

Comment:
This should be within the `force_event_locale` block too - I think we only render stuff from within the macros, but IMHO it's cleaner to keep everything inside..

Reply:
There's two other occurrences of this in the codebase, should I change those as well (in a separate PR)?

Reply:
or just give me a commit to cherrypick... seems like the kind of thing not really worth a PR